File: py/Soal-2/paus.py
from mamalia import Mamalia

class Paus(Mamalia) :
    def __init__(self) :
        super().__init__()
        super().setNama("Paus")
        super().setUkuran("Besar")
        super().setJumlahKaki(0)
        super().setHabitat("Laut")

File: cpp/Soal-2/paus.cpp
#include "mamalia.cpp"
#pragma once

class paus: public mamalia
{
public:
    paus() {
        mamalia::setNama("Paus");
        mamalia::setUkuran("Besar");
        mamalia::setJumlahKaki(0);
        mamalia::setHabitat("Laut");
    }
};